clang/docs/LanguageExtensions.rst

```diff
@@ -745,12 +745,12 @@ Let ``VT`` be a vector type and ``ET`` the element type of ``VT``.
 ======================================= ====================================================================== ==================================
          Name                            Operation                                                              Supported element types
 ======================================= ====================================================================== ==================================
- ET __builtin_reduce_max(VT a)           return x or y, whichever is larger; If exactly one argument is         integer and floating point types
-                                         a NaN, return the other argument. If both arguments are NaNs,
-                                         fmax() return a NaN.
- ET __builtin_reduce_min(VT a)           return x or y, whichever is smaller; If exactly one argument           integer and floating point types
-                                         is a NaN, return the other argument. If both arguments are
-                                         NaNs, fmax() return a NaN.
+ ET __builtin_reduce_max(VT a)           return the largest element of the vector. If the element type is       integer and floating point types
+                                         floating point, this function has the same comparison semantics as 
+                                         ``__builtin_reduce_maximum``.
+ ET __builtin_reduce_min(VT a)           return the smallest element of the vector. If the element type is      integer and floating point types
+                                         floating point, this function has the same comparison semantics as 
```

Comment:
Looking at the implementation, this is going to use reduce_fmax intrinsic, which is not the same as reduce_maximum

I think this builtin trying to support floating point is an anti-feature, and if it has to, it should have matched minimum/maximum. But that's not what the implementation is doing